VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -976,6 +958,31 @@ export abstract class WaitingRoomBaseDirective {
         this.toggleVideoStreamMute(false);
     }
 
+    private handleParticipantsUpdatedMessage(participantsUpdatedMessage: ParticipantsUpdatedMessage) {
+        this.logger.debug(`[WR] - Participants updated message recieved`, participantsUpdatedMessage.participants);
+
+        if (!this.validateIsForConference(participantsUpdatedMessage.conferenceId)) {
+            return;
+        }
```

Comment:
This is the change. I moved into a function to match others and to make it more obvious where this is handled.